mount/mount.c

```diff
@@ -149,8 +173,14 @@ int main() {
 		fprintf(stderr, "titus-mount: using these nfs mount options: %s\n", final_options);
 	}
 
-	/* We don't check for overflow */
-	rc = mount(source, target, fs_type, flags_ul, final_options);
+	if (nfs_mount_hostname && check_new_mount_api_available()) {
```

Comment:
why check nfs_mount_hostname?

Reply:
Hmm. I guess I thought that titus-mount did more than nfs, and that I only wanted the new behavior to be on that.

Looking at the executor code, it looks like we only invoke titus-mount on efs mounts, so this gate won't do anything in practice.